tools/user_migration/migrate_user.py

```diff
@@ -193,6 +199,19 @@ def get_fxa_id(databases, user):
     return (fxa_kid, fxa_uid)
 
 
+def create_migration_table(database):
+    try:
+        cursor = database.cursor()
+        cursor.execute(
+            """create table if not exists migration (
+                fxa_uid varchar(255) NOT NULL PRIMARY KEY,
+                started BIGINT
```

Comment:
probably want NOT NULL?

Reply:
I was debating this. To be honest, we only really need some ID for the user. I figured that "started" might be useful for some diagnostics. Still, adding `NOT NULL` is harmless since we're always setting it to `int(time.time())` since it doesn't need to be precise either.